index.bs

```diff
@@ -892,8 +883,12 @@ user consent to a specific transaction. The structure of these signatures is def
 
 ## Additional options for Assertion Generation (dictionary <dfn dictionary>AssertionOptions</dfn>) ## {#assertion-options}
```

Comment:
Good call. Done in https://github.com/w3c/webauthn/pull/398/commits/0039a138e2d27567e7d32f0ddd203748bedb9cbb.